RedpandaStats/kafka-produce.py

```diff
@@ -0,0 +1,107 @@
+# Example scrupt to produce kafka messages to Deephaven from docker stats using RedPanda.
+#
+# To run this script, you need confluent-kafka libraries installed.
+# To create a dedicated venv for it, you can do:
+#
+# $ mkdir confluent-kafka; cd confluent-kafka
+# $ python3 -m venv confluent-kafka
+# $ cd confluent-kafka
+# $ source bin/activate
+# $ pip3 install confluent-kafka
+#
+# Note: On a Mac you may need to install the librdkafka package.
+# You can use "brew install librdkafka" if the pip3 command fails
+# with an error like "librdkafka/rdkafka.h' file not found"
+# as found at confluentinc/confluent-kafka-python#166.
+#
+# Examples of use for DH testing together with web UI.
+#
+# == Common to all:
+#
+#  * Start the redpanda compose: (cd RedpandaStats && docker-compose up --build)
+#
+# == Example
+#
+# From [http://localhost:10000/ide](http://localhost:10000/ide/):
+# from deephaven import KafkaTools as kt
+# from deephaven import Types as dht
+# result= kt.consumeToTable({'bootstrap.servers': 'redpanda:29092'} , 'docker-stats', key=kt.IGNORE, value=kt.json([
+#    ('container', dht.string),
+#    ('name',   dht.string),
+#    ('cpu',  dht.string),
```

Comment:
Do we need these columns to have a more meaningful types than `string` to be able to do interesting things with them? (eg average, sums across containers, whatever)

Reply:
I changed the ones that make sense to change. There is logic to change others (memoryUsage, memoryLimit, networkInput, networkOutput, blockInput, blockOutput), but then we have unit conversion to worry (ie B vs kB vs MB and GB and GiB vs MiB)